spring-batch-core/src/test/java/org/springframework/batch/core/repository/dao/AbstractJobExecutionDaoTests.java

```diff
@@ -194,14 +194,23 @@ public void testGetMissingLastExecution() {
 	@Transactional
 	@Test
 	public void testFindRunningExecutions() {
-
+		//Normally completed JobExecution as EndTime is populated
```

Comment:
I would add a start time here as it makes no sense to have an execution with an end time but without a start time.  For example:

```java
JobExecution exec = new JobExecution(jobInstance, jobParameters);
exec.setCreateTime(new Date(0));
exec.setStartTime(new Date(1L));
exec.setEndTime(new Date(2L));
```